arangod/GeneralServer/CommTask.cpp

```diff
@@ -127,8 +102,43 @@ bool resolveRequestContext(application_features::ApplicationServer& server,
   // the "true" means the request is the owner of the context
   return true;
 }
+
+bool queueTimeViolated(GeneralRequest const& req) {
+  // check if the client sent the "x-arango-queue-time-seconds" header
+  bool found = false;
+  std::string const& queueTimeValue = req.header(StaticStrings::XArangoQueueTimeSeconds, found);
+  if (found) {
+    // yes, now parse the sent time value
+    double requestedQueueTime = StringUtils::doubleDecimal(queueTimeValue);
```

Comment:
What is the outcome if queueTimeValue is not a valid double? If I understand it will just get ignored, would it make sense signal/inform here (e.g., at least on higher  log-levels)?

Reply:
Understood, this is set by ArangoDB and not user specified